lib/ruby_lsp/requests/semantic_highlighting.rb

```diff
@@ -172,37 +173,59 @@ def visit_kw(node)
       def visit_params(node)
         node.keywords.each do |keyword,|
           location = keyword.location
-          add_token(location_without_colon(location), :variable)
+          add_token(location_without_colon(location), :parameter)
         end
 
         node.requireds.each do |required|
-          add_token(required.location, :variable)
+          add_token(required.location, :parameter)
         end
 
         rest = node.keyword_rest
-        return if rest.nil? || rest.is_a?(SyntaxTree::ArgsForward)
+        unless rest.nil? || rest.is_a?(SyntaxTree::ArgsForward)
```

Comment:
Nitpick: This conditional would be so much easier to read if flipped:
```suggestion
        if rest && !rest.is_a?(SyntaxTree::ArgsForward)
```